posts/2021-04-12-async-vision-doc-shiny-future.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: "Brainstorming Async Rust's Shiny Future"
+author: Niko Matsakis
+description: "Brainstorming Async Rust's Shiny Future"
+team: the Async Foundations Working Group <https://rust-lang.github.io/wg-async-foundations/>
+---
+
+On March 18th, we [announced the start of the Async Vision Doc][announce] process. Since then, we've landed [17 "status quo" stories][sq] and we have [5 more stories in open PRs][prs]; [Ryan Levick] and [I] have also hosted more than ten [collaborative][] [writing][] [sessions][] over the course of the last few weeks.
+
+[collaborative]: https://smallcultfollowing.com/babysteps/blog/2021/03/22/async-vision-doc-writing-sessions/
+[writing]: https://smallcultfollowing.com/babysteps/blog/2021/03/25/async-vision-doc-writing-sessions-ii/
+[sessions]: https://smallcultfollowing.com/babysteps/blog/2021/03/29/async-vision-doc-writing-sessions-iii/
+
+Now that we have a good base of "status quo" stories, we are starting to imagine what the [✨ "shiny future" ✨][sf] might look like. **We want your help!** If you have a great idea for Async Rust[^youknow], then [take a look at the template and open a PR][template]! Alternatively, if you have an idea for a story but would like to discuss it before writing, you can [open a "shiny future" issue][open]. Also, we would still love to get more ["status quo" stories][sq], so please continue to share those.
+
+[^youknow]: Don't be modest. You know you do.
+
+When writing "shiny future" stories, the goal is to focus on the **experience** of Rust's users first and foremost, and not so much on the specific technical details. In fact, you don't even have to know exactly how the experience will be achieved. We have a few years to figure that out, after all. 🚀
+
+Every "shiny future" story is a "retelling" of one or more "status quo" stories. The idea is to replay the same scenario but hopefully with a happier ending, as a result of the improvements we've made. If you don't see a "status quo" story that is right for telling your "shiny future" story, though, that's no problem! Just write it now! We are still requesting "status quo" and "shiny future" stories, and we will do so right up until the end. 
```

Comment:
I think clarifying the exact process here might be helpful- say, if I have a vision for the future, do I need to go dig through the status quo stories? There's already 17, and I imagine more in the pipeline - not everyone will have time to read through all of the existing ones. Maybe we can encourage people to file a shiny future issue, and then work from there both on the async foundations wg side and with the issue filer to find or write a good status quo story?